manifests/init.pp

```diff
@@ -119,27 +119,39 @@
         }
         default:{
           case $version {
-            /^1\.8.*$/:{
+            /^1\.8(\..*)*$/:{
               $real_ruby_package  = "${ruby::params::ruby_package}1.8"
               if ! $suppress_warnings and versioncmp($::operatingsystemrelease, '14.04') >= 0 {
                 warning('Packages for Ruby 1.8 are not available from default repositories.')
               }
             }
-            /^1\.9.*$/:{
+            /^1\.9(\..*)*$/:{
```

Comment:
Again, I'd be more inclined to change the other files to match what's currently in my PR - dev.pp is the only other place this happens.